projects/v2/src/app/store/ui.state.ts

```diff
@@ -375,6 +383,44 @@ export class UIState {
 
   }
 
+  /**
+   * Action to open bottom sheet. Accept the data (name of structure)
+   * First close the bottom sheet, incase it is open.
+   * Then dispatch new action to update bottom sheet data
+   */
+   @Action(OpenBottomSheetDOI)
+   OpenBottomSheetDOI({ getState, setState, dispatch }: StateContext<UIStateModel>, { data }: OpenBottomSheetDOI) {
+     const state = getState();
+     dispatch(new CloseBottomSheet());
+     dispatch(new CloseBottomSheetDOI());
+     dispatch(new UpdateBottomSheetData(data)).subscribe(_ => {
+       setState({
+         ...state,
+         bottomSheetDOIOpen: true
+       });
+     });
+     dispatch(new UpdateBottomSheetDOI(data));
+
+   }
+
+  /**
+   * Action to close bottom sheet.
+   * Empty the bottom sheet data from the state
+   * Set the bottom sheet open variable to false
+   */
+   @Action(CloseBottomSheetDOI)
+   closeBottomSheetDOI({ getState, setState, dispatch }: StateContext<UIStateModel>) {
+     const state = getState();
+     dispatch(new UpdateBottomSheetData({}));
+     const view = this.store.selectSnapshot(TreeState.getVegaView);
+     if (Object.entries(view).length) { view.signal('path__click', {}); }
```

Comment:
What is happening here?

Reply:
I forgot to remove this, Will remove this as well